### PR TITLE
Update to 0.4.10

### PR DESCRIPTION
**Pull request by maximlt**
spatialpandas 0.4.10

**Destination channel:** defaults

### Links

- [Upstream repository](https://github.com/holoviz/spatialpandas)
- [Upstream changelog/diff](https://github.com/holoviz/spatialpandas/releases/tag/v0.4.10)

### Explanation of changes:

- Same as https://github.com/conda-forge/spatialpandas-feedstock/pull/17

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 666c7b33fe0e8516b88029beaf23f7260375ec42997e5a2e40b3e689d73f28cf
+  sha256: 032e24ebb40f75c5c79cb79d7c281f2990e69ba382c0b24acb53da7bba60851c
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: `skip: True  #[py<39 or s390x]`
see https://github.com/holoviz/spatialpandas/blob/v0.4.10/setup.py#L56

Comment (skupr-anaconda) [reply recipe/meta.yaml:13]: Is there missing `pyct` in `host`? see https://github.com/holoviz/spatialpandas/blob/b5da5119a41a200c9a0209b472838c75b0ff81e1/pyproject.toml#L4

Comment (skupr-anaconda) [reply recipe/meta.yaml:13]: I think you can safely replace `param 1.13.0` by `param`

Comment (maximlt) [reply recipe/meta.yaml:13]: > skip: True  #[py<39 or s390x]

Thanks, I missed that!

> Is there missing pyct in host?

True! I added it.

> I think you can safely replace param 1.13.0 by param

Instead I decided to update the host dependencies so they match the build requirements as pinned in `pyproject.toml`. Let me know what you think of this approach.

Comment (maximlt) [reply recipe/meta.yaml:13]: > Instead I decided to update the host dependencies so they match the build requirements as pinned in pyproject.toml. Let me know what you think of this approach.

Ignore my comment, the linter complained, I removed all the pins in the host section.

**Review by skupr-anaconda — APPROVED** [review]

**Review by SC426 — APPROVED** [review]

**Pull request merged**